extension/awsxrayproxy/factory.go

```diff
@@ -43,17 +45,19 @@ func NewFactory() component.ExtensionFactory {
 func createDefaultConfig() config.Extension {
 	return &Config{
 		ExtensionSettings: config.NewExtensionSettings(config.NewID(typeStr)),
-		TCPAddr: confignet.TCPAddr{
-			Endpoint: defaultEndpoint,
-		},
-		ProxyAddress: "",
-		TLSSetting: configtls.TLSClientSetting{
-			Insecure:   false,
-			ServerName: "",
+		ProxyConfig: proxy.Config{
+			TCPAddr: confignet.TCPAddr{
+				Endpoint: defaultEndpoint,
+			},
+			ProxyAddress: "",
+			TLSSetting: configtls.TLSClientSetting{
+				Insecure:   false,
+				ServerName: "",
+			},
+			Region:      "",
+			RoleARN:     "",
+			AWSEndpoint: "",
```

Comment:
Are these all the zero value for their type?  Do they need to be included explicitly here?

Reply:
Thanks, cleaned that up